tests/zfs-tests/tests/functional/mmp/mmp_hostid.ksh

```diff
@@ -64,10 +67,11 @@ log_must mv $HOSTID_FILE $mntpnt_etc/hostid
 # 3. Create a file so the pool will have some contents
 log_must zfs create $MMP_POOL/fs
 mntpnt_fs=$(get_prop mountpoint $MMP_POOL/fs)
-log_must mkfile 1M $fs_mntpnt/file
+log_must mkfile 1M $mntpnt_fs/file
 
 # 4. Verify multihost cannot be enabled until the /etc/hostid is linked
 log_mustnot zpool set multihost=on $MMP_POOL
+log_mustnot ls -l $HOSTID_FILE
```

Comment:
This entire test group is normally skipped when a `$HOSTID_FILE` exists.  See the check in `zfs-tests/tests/functional/mmp/setup.ksh`.  Though replicating that check before the test even starts seems like a good idea.

```sh
 if [ -e $HOSTID_FILE ]; then
        log_unsupported "System has existing $HOSTID_FILE file"
fi
```

Reply:
this check before `ln -s`, because file was moved, but in some way - probably by parallel task, can be restored and `ln -s` will do some mistake